docusign/config/index.js

```diff
@@ -0,0 +1,41 @@
+exports.docOptions = require('./documentOptions.json');
```

Comment:
ESM is not supported? CommonJS currently looks to be deprecated.

Reply:
The DocuSign starter project was coded in CommonsJS. It would take a fair amount of work to convert it to ESM.

AFAIK CommonJS is not "deprecated", though I agree that it should not be used for new projects.

Reply:
Yeah, "deprecated" is a misnomer, "unrecommended" is more appropriate. Since DocuSign itself uses this approach, let's leave it as it is.